lib/src/main/java/org/openapitools/openapistylevalidator/ErrorAggregator.java

```diff
@@ -5,15 +5,15 @@
 import org.eclipse.microprofile.openapi.models.PathItem;
 import org.openapitools.openapistylevalidator.styleerror.*;
 
-class ErrorAggregator {
+public class ErrorAggregator {
```

Comment:
This class should not be public, it can only be used in this module. Was there a reason for this?

Reply:
Makes sense!